test/SerialIOTest.cpp

```diff
@@ -5856,21 +5864,21 @@ void variableBasedSeries(std::string const &file)
             // this tests changing extents across iterations
             // ADIOS2 does not support changing the dimensionality
             // (older versions used to somewhat support it, but not really)
-            auto E_y = iteration.meshes["E"]["y"];
+            auto B_y = iteration.meshes["B"]["y"];
```

Comment:
Interesting example. I was thinking primarily about particle records, but yes fields also could be used to shapes of some components depend on others...